blocksync/blocksync.go

```diff
@@ -42,7 +43,7 @@ type (
 	// BlockSync defines the interface of blocksyncer
 	BlockSync interface {
 		lifecycle.StartStopper
-
+		nodestats.StatsReporter
```

Comment:
remove this, don't need to explicitly tell, so we can remove the import at L26 above